.circleci/config.yml

```diff
@@ -36,35 +36,69 @@ jobs:
 workflows:
   ci:
     jobs:
+      - bundle_and_test:
+          name: ruby2-7_rails6-0
+          ruby_version: 2.7.0
+          rails_version: 6.0.2
+      - bundle_and_test:
+          name: ruby2-6_rails6-0
+          ruby_version: 2.6.5
+          rails_version: 6.0.2
+      - bundle_and_test:
+          name: ruby2-5_rails6-0
+          ruby_version: 2.5.7
+          rails_version: 6.0.2
+      - bundle_and_test:
+          name: ruby2-4_rails6-0
+          ruby_version: 2.4.6
+          rails_version: 6.0.2
+      - bundle_and_test:
+          name: jruby9-2_rails6-0
+          ruby_type: jruby
+          ruby_version: 9.2.9.0
+          rails_version: 6.0.2
+      - bundle_and_test:
+          name: jruby9-1_rails6-0
+          ruby_type: jruby
+          ruby_version: 9.1.17.0
+          rails_version: 6.0.2
```

Comment:
It looks like jruby 9.2.x is compatible with ruby 2.5.x so jruby 9.1 can be dropped as not supported by rails 6.